### PR TITLE
Documents projectId field

### PR DESCRIPTION
**Pull request by albertmourato**
Closes https://github.com/wedeploy/wedeploy.com/issues/607

### CODE REVIEW

File: node/electric/src/pages/docs/deploy/configuring-deployments/index.md [hunk 1]
```diff
@@ -261,6 +262,19 @@ In this example, the `photos` volume will be shared and both services can access
 
 **Note:** Once you delete your project, any files in the volumes will also be destroyed.
 
+<h4 id="projectId">projectId</h4>
+
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/deploy/configuring-deployments/index.md]: We need to describe here the primary benefit of using this field - to deploy a project to the same project without having to declare it on the CLI every time you deploy.

File: node/electric/src/pages/docs/deploy/configuring-deployments/index.md [hunk 2]
```diff
@@ -261,6 +262,19 @@ In this example, the `photos` volume will be shared and both services can access
...
+To create a new project, you have to choose its unique `projectId`. It can be defined on `wedeploy.json`.
+
+```application/json
+{
+  "id": "ui",
+  "projectId": "myproject"
+}
+```
+
+You can also choose your `projectId` when creating a new project by performing `we new -p myproject` from your [command line](https://wedeploy.com/docs/intro/using-the-command-line/).
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/deploy/configuring-deployments/index.md]: Here we could say something like this:

"If you would rather not have a hardcoded Project ID value, you can pass the ID when you run the deploy command from the CLI"

**Comment by albertmourato** [comment]
@jonnilundy take a look now. If you find something that I can improve, please let me know.

**Comment by albertmourato** [comment]
Duplicated of https://github.com/wedeploy/wedeploy.com/pull/689

**Pull request closed without merging**